{
  "Header": {
    "Original": "CVE-2022-40898 (An issue discovered in Python Packaging Authority (PyPA) Wheel 0.37.1  ...)",
    "Line": 53697,
    "ID": "CVE-2022-40898",
    "Description": "(An issue discovered in Python Packaging Authority (PyPA) Wheel 0.37.1  ...)"
  },
  "Annotations": [
    {
      "Original": "- wheel 0.38.0-1 (unimportant)",
      "Line": 53698,
      "Type": "package",
      "Package": "wheel",
      "Kind": "fixed",
      "Version": "0.38.0-1",
      "Severity": "unimportant"
    },
    {
      "Original": "NOTE: https://github.com/pypa/wheel/commit/88f02bc335d5404991e532e7f3b0fc80437bf4e0",
      "Line": 53699,
      "Type": "NOTE",
      "Description": "https://github.com/pypa/wheel/commit/88f02bc335d5404991e532e7f3b0fc80437bf4e0"
    },
    {
      "Original": "NOTE: https://github.com/pypa/wheel/issues/498",
      "Line": 53700,
      "Type": "NOTE",
      "Description": "https://github.com/pypa/wheel/issues/498"
    },
    {
      "Original": "NOTE: Negligible security impact",
      "Line": 53701,
      "Type": "NOTE",
      "Description": "Negligible security impact"
    }
  ]
}